### PR TITLE
Feat: Zetachain PumpBTC Market, Taiko USDa Market

### PR DESCRIPTION
**Pull request by jasmin-avalon**
Add new markets TVL to Avalon Labs DeFi Lending: 
- Zetachain PumpBTC Market: https://app.avalonfinance.xyz/?marketName=proto_zetachain_pump_btc_innovation
- Taiko USDa Market: https://app.avalonfinance.xyz/?marketName=proto_taiko_usda_innovation

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/avalon-finance exports TVL: 
        
 
 
    core                      255.37 M
    btr                       254.91 M
    borrowed                  210.62 M
    taiko                     169.01 M
    bsc                       111.16 M
    merlin                    87.34 M
    bob                       75.22 M
    btr-borrowed              67.71 M
    core-borrowed             67.49 M
    bsc-borrowed              66.31 M
    mode                      43.36 M
    merlin-borrowed           5.74 M
    zeta                      4.71 M
    arbitrum                  3.94 M
    bob-borrowed              2.31 M
    ethereum                  2.01 M
    iotex                     1.87 M
    arbitrum-borrowed         956.62 k
    base                      266.64 k
    base-borrowed             68.19 k
    ethereum-borrowed         28.16 k
    scroll                    11.32 k
    bsquared                  10.94 k
    iotex-borrowed            2.48 k
    klaytn                    1.12 k
    klaytn-borrowed           1.08 k
    bsquared-borrowed         296
    zeta-borrowed             39
    scroll-borrowed           0
    mode-borrowed             0
    taiko-borrowed            0
    
    total                    1.01 B

**Comment by jasmin-avalon** [comment]
The **Taiko USDa Market** is only under DeFi Lending, not in USDaLend, so there's no double-counting issue.

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
you requested for a new sub-listing for USDa lending, then why add it again (iotex is duplicated atm) here? Can you move the taiko part to that adapter?

**Comment by g1nt0ki** [comment]
> The **Taiko USDa Market** is only under DeFi Lending, not in USDaLend, so there's no double-counting issue.

but iotex part is?

**Comment by jasmin-avalon** [comment]
Hey hey @g1nt0ki 

That taiko-USDa market is not duplicated. The USDaLend on IoTeX is a fixed rate lending with our own POL. And we only count the USDa on IoTeX chain. You can see the Borrow APR is fixed at around 21%. https://usdalend.avalonfinance.xyz

The taiko USDa pool is just a classic lending. The rates totally depend on market demand. And we only count the USDa on taiko. https://app.avalonfinance.xyz/?marketName=proto_taiko_usda_innovation

So I don’t think there is double counting issue.

File: projects/avalon-finance/index.js [hunk 1]
```diff
@@ -74,7 +75,14 @@ const listaMarkets = {
 	bsc: aaveExports('', '', undefined, ['0x5157f63bE7808DEB090Eee7762e917745896A09E'], { v3: true }), // BSC - ListaDAO
 }
 
+// @note USDa Defi Lending Markets, not USDaLend
+const usdaDefiLendingMarkets = {
+	iotex: aaveExports('', '', undefined, ['0xDB52DD393e3a5e95d3B7C7e1C42cC06bb807A369'], { v3: true }),
```
Comment (g1nt0ki) [inline projects/avalon-finance/index.js:80]: @jasmin-avalon I meant this line

**Comment by llamatester** [comment]
The adapter at projects/avalon-finance exports TVL: 
        
 
 
    core                      251.00 M
    btr                       250.74 M
    borrowed                  207.25 M
    taiko                     165.99 M
    bsc                       109.12 M
    merlin                    85.17 M
    bob                       73.97 M
    btr-borrowed              66.59 M
    core-borrowed             66.30 M
    bsc-borrowed              65.09 M
    mode                      42.59 M
    merlin-borrowed           5.65 M
    zeta                      4.71 M
    arbitrum                  3.90 M
    bob-borrowed              2.56 M
    ethereum                  1.97 M
    arbitrum-borrowed         968.69 k
    base                      249.82 k
    iotex                     215.47 k
    klaytn                    159.81 k
    base-borrowed             66.92 k
    ethereum-borrowed         27.64 k
    scroll                    11.22 k
    bsquared                  10.79 k
    iotex-borrowed            2.49 k
    klaytn-borrowed           1.06 k
    bsquared-borrowed         292
    zeta-borrowed             40
    mode-borrowed             0
    scroll-borrowed           0
    taiko-borrowed            0
    
    total                    989.79 M

File: projects/avalon-finance/index.js (hunk 1 above)
Comment (jasmin-avalon) [reply projects/avalon-finance/index.js]: Oh, the IoTex-USDa does a duplicate one. I've removed it in the latest commit(https://github.com/DefiLlama/DefiLlama-Adapters/pull/12465/commits/55ad0917a34c8f896fd31fc8b409cae48f58a14d). 
We just want to add the Zetachain-PumpBTC and Taiko-USDa to DeFi Lending.  
I think it should be good now.

**Pull request merged**